internal/nodes/dequeue_ingestion.go

```diff
@@ -508,16 +508,28 @@ func handleL4SharedVipService(namespacedVipKey, key string, fullsync bool) {
 		// Initializing the preferred VIP from the first Service we get, so any other Service
 		// that wishes for static IP allocation differently conflicts with this.
 		if i == 0 {
-			sharedVipLBIP = svcObj.Spec.LoadBalancerIP
+			if svcObj.Spec.LoadBalancerIP != "" {
```

Comment:
these conditions are repeated everywhere. It will be better if we can move this to a function.

Reply:
Added functions in util.go to handle condition checks